tools/wheels/cibw_test_command.sh

```diff
@@ -26,6 +30,9 @@ fi
 # Set available memory value to avoid OOM problems on aarch64.
 # See gh-22418.
 export NPY_AVAILABLE_MEM="4 GB"
+if [[ $(python -c"import sysconfig; print(bool(sysconfig.get_config_var('Py_GIL_DISABLED')))") == "True" ]]; then
+   export PYTHON_GIL=0
```

Comment:
maybe add a TODO to remove this when numpy's extension modules no longer disable the GIL on import?

Reply:
+1 for the TODO with a comment descriptive enough that someone who doesn't know anything about the free-threaded build will know what the env var is for